### ISSUE TITLE
simpleJSON install breaks distributed version of web2py

### ISSUE THREAD

**Issue by henkelis** [henkelis/sonospy#54]
If simpleJSON is installed (for example in Fedora 15 or macports), the version of web2py distributed with Sonospy won't run. This is fixed in a more recent version of web2py.

**Comment by michaelnt** [comment]
Is this the error that is displayed (running on Ubuntu 11.04 with simplejson 2.1.2)

AttributeError: 'JSONDecoder' object has no attribute 'object_pairs_hook'

**Comment by henkelis** [comment]
Yes. You can overcome it by installing the latest version of web2py. 
Note that the web2py folder in sonospy also contains the sonospy web2py 
app (sonospy/web2py/applications/sonospy/*), so you need to copy that 
back into the newly installed web2py install. You also need to copy in 
options.py from the web2py folder (that contains sonospy specific config).
